src/fights.ts

```diff
@@ -750,6 +751,16 @@ const freeFightSources = [
     () => withMacro(Macro.basicCombat(), () => use($item`molehill mountain`)),
     true,
   ),
+  new FreeFight(
```

Comment:
I personally would have done a like `...getBestScepterSkills().map(({ skill }) => ` whatever, but given that we're only supporting 2 fights this is fine